test/TestApps/AppLifecycleTestApp/Helpers.cpp

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include "Helpers.h"
+#include <TestDef.h>
+
+const wchar_t* g_bootStrapDllName = L"Microsoft.ProjectReunion.Bootstrap.dll";
+
+typedef HRESULT (*BootStrapTestInit)(PCWSTR prefix, PCWSTR publisherId);
+typedef HRESULT (*BootStrapInit)(const UINT32 majorMinorVersion, const PACKAGE_VERSION minVersion);
```

Comment:
>rVersion, co [](http://example.com/codeflow?start=55&length=12)

MddBootstrapInitialize() recently changed to add a 3rd parmeter -- majorminor, PCWSTR versionTag, minversion

Needed to support Reunion's versioning. Pass null or "" for 'no tag' and you're good

Alas, it's a C API so I couldn't overload it to accept 2 + 3 parameters and make versionTag optional